### ISSUE TITLE
Check for trailing comma in last line of a attribute/value list

### ISSUE THREAD

**Issue by sebastien-prudhomme** [rodjek/puppet-lint#237]
Not in the official Puppet style guide (but used to be in an old official Puppet best practice wiki page)

Good:

    exec { 'blah':
      path => '/usr/bin',
      cwd  => '/tmp',
    }

Bad:

    exec { 'blah':
      path => '/usr/bin',
      cwd  => '/tmp'
    }

**Comment by flypenguin** [comment]
The funny thing is: It does not complain about actual wrong commas.
This triggers a complaint:
```
class { 'thing':
  one   => two,
  three => four
}
```
... whereas this doesn't:
```
class { 'thing':
  one   => two
  three => four,
}
```
which strikes me as ... weird? Or do I not see something here?

**Comment by ghoneycutt** [comment]
@flypenguin The first is a style issue whereas the second is a syntax issue, ie: the first is acceptable to the Puppet language and the second cannot be parsed.

**Comment by daenney** [comment]
So, this already exists now as a puppet-lint plugin here: https://github.com/puppet-community/puppet-lint-trailing_comma-check.

Time to close this issue.
